### PR TITLE
added more starting positions

### PR DESCRIPTION
**Pull request by Jackack**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description
added more starting positions
added 15 second wait before short circuit moves -> this is required for MPC to properly initialize
increased sim noise

## QA Instructions, Screenshots, Recordings

run roslaunch buggy sim_2d_2buggies.launch manual_vel:=true. Set short circuit velocity to 10 immediately. Short circuit position should show up immediately in foxglove, but only starts moving after 15 seconds.

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

- [ ] Yes
- [x] No, and this is why: not easily testable with automated script, but easily testable by just running the sim.
- [ ] I need help with writing tests

### CODE REVIEW

File: rb_ws/src/buggy/scripts/2d_sim/engine.py [hunk 1]
```diff
@@ -227,8 +234,8 @@ def publish(self):
         heading_noisy = heading
 
         if (Simulator.NOISE):
-            lat_noisy = lat + np.random.normal(0, 1e-8)  # ~.1cm error
-            long_noisy = long + np.random.normal(0, 1e-8)  # ~.1cm error
+            lat_noisy = lat + np.random.normal(0, 1e-6)  # ~10cm error
```
Comment (mehulgoel873) [inline rb_ws/src/buggy/scripts/2d_sim/engine.py:237]: If our current error on the actual system is 0.5m, shouldn't the noise be more reflective of that?

Comment (christianvluu) [reply rb_ws/src/buggy/scripts/2d_sim/engine.py:237]: this is Gaussian noise, are you referring to the controller error? if so that is unrelated.

**Review by christianvluu — CHANGES_REQUESTED** [review]
looks bueno

**Review by christianvluu — APPROVED** [review]
looks bueno my bad hit the wrong button

**Pull request merged**